schema.graphql

```diff
@@ -390,6 +390,12 @@ type SystemState @entity {
   " Total savings in DAI of all users (Pot#Pie) "
   totalSavingsInPot: BigDecimal!
 
+  " Value which decides how much dai - given when drip() is called "
+  rateAccumulator: BigInt
+
+  " last time the drip() function was called "
+  lastDripCall: BigInt
+
```

Comment:
I suggest adding the name of the contract to the description and to the var name. There is another drip in the Jug contract.